gradle-jdks-excavator-configurations/src/main/resources/install-jdks.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -eux
+
+die () {
+    echo
+    echo "$*"
+    echo
+    rm -rf "$tmp_work_dir"
+    exit 1
+} >&2
+
+read_value() {
+  if [ ! -f "$1" ]; then
+    die "ERROR: $1 not found, aborting Gradle JDK installation"
+  fi
+  read -r value < "$1" || die "ERROR: Unable to read value from $1. Make sure the file ends with a newline."
+  echo "$value"
+}
+
+GRADLE_JDKS_DIR=$1
+CERTS_DIR=$2
+
+if [[ -z "$CERTS_DIR" || -z "$GRADLE_JDKS_DIR" ]]; then
+  die "ERROR: Missing argument CERTS_DIR"
+fi
```

Comment:
This will never happen as due to `-u` the `CERTS_DIR=$1` will explode (and `-e` will cause the script to exit immediately).